src/forms/BoundDateField.tsx

```diff
@@ -13,12 +16,14 @@ export type BoundDateFieldProps = Omit<DateFieldProps, "value" | "onChange" | "o
 /** Wraps `TextField` and binds it to a form field. */
 export function BoundDateField(props: BoundDateFieldProps) {
   const { field, onChange = (value) => field.set(value), label = defaultLabel(field.key), ...others } = props;
+  const settings = useContext(FormContext);
   const testId = useTestIds(props, field.key);
   return (
     <Observer>
       {() => (
         <DateField
           label={label}
+          labelSuffix={getLabelSuffix(settings, field)}
```

Comment:
Teaching the bound fields to auto-add `First Name (Required)` where the "Required" is only added if the `FormLines` parent opts-in to a given label suffix style.

Personally, I'd prefer to have this hard-coded, like we _always_ put `(Required)` and/or `(Optional)` after field lines, but I'm anticipating design wanting more flexibility.